packages/backend/server/src/modules/sync/events/events.gateway.ts

```diff
@@ -175,7 +266,10 @@ export class EventsGateway implements OnGatewayConnection, OnGatewayDisconnect {
     },
     @ConnectedSocket() client: Socket
   ) {
-    if (!client.rooms.has(workspaceId)) {
+    if (
+      !client.rooms.has(workspaceId) &&
```

Comment:
it's ok to remove `workspaceId` room in server because all users are gonna reconnect if when deploying.

Reply:
removed